include/glow/Importer/CommonOperatorLoader.h

```diff
@@ -437,9 +434,18 @@ class CommonOperatorLoader : public ProtobufLoader {
       return true;
     }
     if (typeName == "Dropout") {
-      loadDropout(op, dict);
+    // Save the identity operation. Note the second output (mask) for Dropout in
```

Comment:
also, make sure to run clang-format.

Reply:
yes, i had some issues wrt to the version of clang-format I was using, I will rerun with v6.0